### ISSUE TITLE
No feedback on progress while marking messages as spam

### ISSUE THREAD

**Issue by skandragon** [modoboa/modoboa-amavis#77]
While marking messages as spam, there is no feedback that it's doing anything, causing the user to wonder if they clicked or not, and usually clicking again.

**Comment by tonioo** [comment]
Running the learning process synchronously is a pretty bad idea but it was quick to implement... It runs fine for 1/2 messages but that's all. An asynchronous mechanism would be better but it would require to introduce new dependencies to Modoboa (such as RQ or Celery)...

**Comment by skandragon** [comment]
Agreed, it's hard, but right now the experience is it doesn't really work, so...

**Comment by tonioo** [comment]
It would also require some work... Do you manage a big mail server?

**Comment by skandragon** [comment]
No, but even marking a few tens of messages takes many minutes, while the UI makes it unclear if it's even started or not.  I'm OK with it just saying "OK, you clicked the button correctly, we're working on this..." rather than no visible UI changes.
